static/sass/_patterns_social.scss

```diff
@@ -0,0 +1,41 @@
+@mixin p-social {
+  .p-social-list {
+    @extend .p-inline-list;
+    margin-top: 1rem;
```

Comment:
If the only reason to introduce `p-social-list` is changing the top margin maybe it would be nice to have means to control margin from vanilla itself?

Reply:
I do agree it feels kinda superfluous. Maybe it would be neater to implement our own utility class....